src/Security/Security/Models/SecurityContact/PSSecurityContact.cs

```diff
@@ -24,8 +26,8 @@ public class PSSecurityContact
 
         public string Phone { get; set; }
 
-        public string AlertNotifications { get; set; }
+        public PSSecurityContactPropertiesAlertNotifications AlertNotifications { get; set; }
 
-        public string AlertsToAdmins { get; set; }
+        public PSSecurityContactPropertiesNotificationsByRole NotificationsByRole { get; set; }
```

Comment:
As @Nickcandy mentioned, changing parameter type is regarded as breaking change. It wouldn't be allowed in regualer release. Please revert them. To work around the issue, how about adding two new parameters for these objects?